tasks/check-example.js

```diff
@@ -0,0 +1,65 @@
+/*eslint-env es6*/
```

Comment:
We could add `tasks/.eslintrc` with `{"env": {"es6": true}}` to get the same for all tasks.